### ISSUE TITLE
v1.10.12: generated_plugin_registrant.dart: Undefined name 'AnotherFlushbarPlugin'

### ISSUE THREAD

**Issue by eggnstone** [cmdrootaccess/another-flushbar#11]
Upgrading from 1.10.11 to 1.10.12 introduces this error in the Dart Analysis windows.
Compilation works though.

```
error: Undefined name 'AnotherFlushbarPlugin'. (undefined_identifier at [xyz] lib\generated_plugin_registrant.dart:21)
info: Unused import: 'package:another_flushbar/flushbar.dart'. (unused_import at [xyz] lib\generated_plugin_registrant.dart:7)
```

**Comment by Raphael-Jose-Ferraro-IRForever** [comment]
same in here

**Comment by cmdrootaccess** [comment]
fixed in 1.10.13.
